kornia/enhance/normalize.py

```diff
@@ -184,3 +193,46 @@ def denormalize(
     out: torch.Tensor = (data * std) + mean
 
     return out
+
+
+def normalize_min_max(x: torch.Tensor, min_val: float = 0., max_val: float = 1., eps: float = 1e-6) -> torch.Tensor:
```

Comment:
[Commenting after a question abt unions on slack]

You can support a union input type with`torch.jit._overload`. It's used [here](https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L2890) to support a different input type list. 

The one limitation (why it's current `_overload` and not `overload`) is that you can't script the function directly; it has to be scripted from another scripted function.

Reply:
@eellison thanks for your suggestion - I'll give it a shot.

Reply:
@eellison I've been playing with this and not quite sure it an handle our use case.

![image](https://user-images.githubusercontent.com/5157099/97804935-2dba9d00-1c53-11eb-8f70-5adf59d84bce.png)
the error is the following: `RuntimeError: Function __torch__.my_fcn cannot be directly compiled because it is overloaded. It must be used in a context of a function where its inputs can determine which overload to call.`

you can check this colab to see a minimal example for that we want to do
https://colab.research.google.com/drive/1cswmoxLryLwTBfAZG1VgAUTBK_ZfAkQ1?usp=sharing